.gitlab-ci.yml

```diff
@@ -6,7 +6,9 @@ stages:
   - macrobenchmarks
 
 include:
-  - remote: https://gitlab-templates.ddbuild.io/libdatadog/include/one-pipeline.yml
+  - project: DataDog/apm-reliability/libdatadog-build
```

Comment:
This is as good a time as any to put a comment in here linking to the actual repo. It's nearly impossible to find based on the information here alone.